File: 217-contains-duplicate/contains-duplicate.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Boolean
     */
    function containsDuplicate($nums) {
        
        $map = array();
        $count = array_count_values($nums);
        //$val = array_values($count);
        foreach($count as $keys => $values){
            if($values >=2){
                return true;
            }
        }
        print_r($count);
        return false;
    }
}

File: 217-contains-duplicate/contains-duplicate.dart
class Solution {
  bool containsDuplicate(List<int> nums) {
    //List<int> dup = [];
    nums.sort();
    for(int i=0;i<nums.length-1;i++){
        /* if(!dup.contains(nums[i])){
            dup.add(nums[i]);
        }else{
            return true;
        } */
        if(nums[i] == nums[i+1]){
            return true;
        }
    }
    return false;
    
  }
}